src/scenes/Home/components/Weather/WeatherIcon.js

```diff
@@ -43,3 +43,32 @@ export function weatherIcon(weatherId, timeOfDay) {
 
 };
   
+export function weatherForecastIcon(weatherForecastId, timeOfDay) {
+
+    
+    if (weatherForecastId <= 232) {
+        return ThunderStormIcon;
+    } else if (weatherForecastId >= 300 && weatherForecastId <= 399) {
+        return ShowerIcon;
+    } else if (weatherForecastId >= 500 && weatherForecastId <= 599) {
+        return RainIcon;
+    } else if (weatherForecastId >= 600 && weatherForecastId <= 622) {
+        return SnowIcon;
+    } else if (weatherForecastId >= 700 && weatherForecastId <= 799) {
+        return MistIcon;    
+    } else if ((weatherForecastId === 800) && (timeOfDay >= 6 && timeOfDay <= 18)) {
```

Comment:
Try making this not with if else , but you will have an object that will hold the range and name as a key value, and you will have to just loop over that object to get the value needed, this if else is a bit bad code

Reply:
done